Solutions/Authomize/Analytic Rules/Refactor_AWS_policy_based_on_activities.yaml

```diff
@@ -0,0 +1,61 @@
+id: 642de064-c67b-4eb7-98bd-3f8cd51f282c
+name: Refactor AWS policy based on activities in the last 60 days
+kind: Scheduled
+description: This is a recommended update to IAM policy on AWS. Review the policy and apply it according to change control process. Authomize will have a recommended policy to be downloaded.
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- PrivilegeEscalation
+relevantTechniques:
+- T1078
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
```

Comment:
we can use extend columns in single line

Reply:
done.